[2022-12-14T12:31:19.380+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: weekly_wrapped_dag.generate_wrapped manual__2022-12-14T12:31:15.742301+00:00 [queued]>
[2022-12-14T12:31:19.389+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: weekly_wrapped_dag.generate_wrapped manual__2022-12-14T12:31:15.742301+00:00 [queued]>
[2022-12-14T12:31:19.390+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-14T12:31:19.391+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-14T12:31:19.392+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-14T12:31:19.404+0000] {taskinstance.py:1304} INFO - Executing <Task(_PythonDecoratedOperator): generate_wrapped> on 2022-12-14 12:31:15.742301+00:00
[2022-12-14T12:31:19.414+0000] {standard_task_runner.py:55} INFO - Started process 28002 to run task
[2022-12-14T12:31:19.424+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'weekly_wrapped_dag', 'generate_wrapped', 'manual__2022-12-14T12:31:15.742301+00:00', '--job-id', '90', '--raw', '--subdir', 'DAGS_FOLDER/weekly_dag.py', '--cfg-path', '/tmp/tmpmsissmgv']
[2022-12-14T12:31:19.425+0000] {standard_task_runner.py:83} INFO - Job 90: Subtask generate_wrapped
[2022-12-14T12:31:19.513+0000] {task_command.py:389} INFO - Running <TaskInstance: weekly_wrapped_dag.generate_wrapped manual__2022-12-14T12:31:15.742301+00:00 [running]> on host b92abc15fbab
[2022-12-14T12:31:19.577+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weekly_wrapped_dag
AIRFLOW_CTX_TASK_ID=generate_wrapped
AIRFLOW_CTX_EXECUTION_DATE=2022-12-14T12:31:15.742301+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-14T12:31:15.742301+00:00
[2022-12-14T12:31:19.588+0000] {base.py:73} INFO - Using connection ID 'spotify_etl' for task execution.
[2022-12-14T12:31:19.593+0000] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/decorators/base.py", line 217, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/weekly_dag.py", line 58, in generate_wrapped
    "played" : artist[1]
IndexError: string index out of range
[2022-12-14T12:31:19.601+0000] {taskinstance.py:1327} INFO - Marking task as FAILED. dag_id=weekly_wrapped_dag, task_id=generate_wrapped, execution_date=20221214T123115, start_date=20221214T123119, end_date=20221214T123119
[2022-12-14T12:31:19.611+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 90 for task generate_wrapped (string index out of range; 28002)
[2022-12-14T12:31:19.636+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2022-12-14T12:31:19.658+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
